sql/core/src/test/scala/org/apache/spark/sql/SQLInsertTestSuite.scala

```diff
@@ -122,6 +125,16 @@ trait SQLInsertTestSuite extends QueryTest with SQLTestUtils {
     }
   }
 
+  test("insert with column list - by name") {
```

Comment:
I will add doc and more test case later. Before do this I want to make sure my idea without problem.